automated install/basic-install.sh

```diff
@@ -1235,13 +1237,18 @@ installConfigs() {
     echo -e "  ${INFO} Installing configs from ${PI_HOLE_LOCAL_REPO}..."
     # Make sure Pi-hole's config files are in place
     version_check_dnsmasq
-
+    # Install template if it does not exist
```

Comment:
This comment should specify that an empty file is installed (at least for the moment), not a template.